packages/react-icons/src/contexts/IconDirectionContext.ts

```diff
@@ -10,4 +10,4 @@ const IconDirectionContextDefaultValue: IconDirectionContextValue = {};
 
 export const IconDirectionContextProvider = IconDirectionContext.Provider;
 
-export const useIconContext = () => React.useContext(IconDirectionContext) ?? IconDirectionContextDefaultValue;
\ No newline at end of file
+export const useIconContext = () => React.useContext(IconDirectionContext) ? IconDirectionContextDefaultValue : undefined;
```

Comment:
I don't understand this change. This is saying that if the context has a value, use the default value instead; otherwise, use `undefiend`.